CleanWeather/Models/City.swift

```diff
@@ -15,3 +15,9 @@ struct City: Codable, Equatable {
     let latitude: Double
     let longitude: Double
 }
+
+extension Array where Element == City {
+    func sortCitiesByName() -> [City] {
```

Comment:
if you have:
let cities: [City]
it looks bad to have method named sortCITIESByName
this extension works only on [City] so there's no point of having it in method signature name